<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NetProxySettings</class>
 <widget class="QDialog" name="NetProxySettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>292</width>
    <height>288</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>292</width>
    <height>288</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>292</width>
    <height>288</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>0</y>
     <width>281</width>
     <height>185</height>
    </rect>
   </property>
   <property name="title">
    <string>UDP Settings</string>
   </property>
   <widget class="QLabel" name="m_lbl_udp_listen_port">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>121</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>UDP listen port:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="m_sb_udp_port_local">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>65535</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="m_lbl_udp_remote_host">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>UDP remote host:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="m_le_udp_remote_host">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>90</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="m_bt_udp_help">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>150</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>?</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="m_sb_udp_port_remote">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>120</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>65535</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="m_lbl_udp_remote_port">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>131</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Remote UDP port:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="m_cb_udp_local_ip">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="m_lbl_udp_listen_ip">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>UDP listen IP:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="m_btn_udp">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Listen</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>190</y>
     <width>281</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>TCP Server Settings</string>
   </property>
   <widget class="QLabel" name="m_lbl_local_tcp_port">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Local TCP port:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="m_sb_tcp_port_local">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>65535</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QPushButton" name="m_bt_tcp_help">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="m_btn_tcp">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Listen</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
